taker/src/main.rs

```diff
@@ -117,7 +126,19 @@ impl Opts {
             },
         };
 
-        Ok((maker_url, maker_id))
+        let maker_peer_id = match self.maker_peer_id {
+            Some(maker_peer_id) => maker_peer_id,
+            None => match network {
+                // TODO: Add defaults for mainnet and testnet
+                Network::Mainnet { .. } => bail!("No maker default peer id configured for mainnet"),
+                Network::Testnet { .. } => bail!("No maker default peer id configured for testnet"),
+                Network::Signet { .. } => {
+                    bail!("No maker default peer id configured for signet")
+                }
```

Comment:
Signet is shorter than the other two, why does this one wrap?

Reply:
I have no idea, I guess this is a question to `rustfmt` :)